src/H5T.c

```diff
@@ -5217,7 +5217,12 @@ H5T__path_find_real(const H5T_t *src, const H5T_t *dst, const char *name, H5T_co
     } /* end else-if */
 
     /* Set the flag to indicate both source and destination types are compound types
-     * for the optimization of data reading (in H5Dio.c). */
+     * for the optimization of data reading (in H5Dio.c).
+     *
+     * Jan-Willem Blokland, 11 March 2020
```

Comment:
Please remove this line. It's going to be really distracting if the code fills up with line-by-line credit for edits.

Reply:
Good point. I will remove it.